test/test_advisory.py

```diff
@@ -368,3 +369,38 @@ def test_advisory_html_urlize_description(db, client):
     assert 'qux <a href="/{0}">{0}</a> is'.format('AVG-1') in data
     assert 'bar <a href="{0}" rel="noopener">{0}</a> is'.format('https://foo.bar') in data
     assert 'lol <a href="/{0}">{0}</a>.'.format('CVE-1111-2222') in data
+
+
+@pytest.mark.parametrize('patch_get', [404], indirect=True)
```

Comment:
I'm not sure if we need to parametrize/monkeypatch the three functions. If I try to make the get request fail with:

```
diff --git a/test/conftest.py b/test/conftest.py
index 73fdef4..8653016 100644
--- a/test/conftest.py
+++ b/test/conftest.py
@@ -87,6 +87,7 @@ def patch_get(monkeypatch, request):
             status_code = request.param
 
     def mocked_get(uri, *args, **kwargs):
+        raise Exception("wat")
         return type('MockedReq', (), {'status_code': status_code, 'text': text})()
     monkeypatch.setattr(tracker.advisory, 'get', mocked_get)
 
```

Only  `test_advisory_publish_advisory`  fails.  Is this intended?

Reply:
Hmm well:

* test_advisory_publish_advisory_not_found, checks how we handle the 404 in advisory_fetch_from_mailman.
* test_advisory_publish_advisory_text_wrong, checks how we handle an invalid ASA text

So those are both different use cases.